python/ray/_private/metrics_agent.py

```diff
@@ -371,6 +379,18 @@ def to_metric(
             )
             return metric
 
+        elif isinstance(agg_data, SumAggregationData):
```

Comment:
Can you add a comment why we convert Sum -> Counter (basically opencensus sum allows to increment float > 1)